### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by eiriksm**

### CODE REVIEW

**Comment by eiriksm** [comment]
Now tested and works. Also needed the correct bootstrap annotation.

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "description": "Allows you to get values for read only settings that are stored in settings.php",
     "type": "drupal-drush",
     "require": {
-        "drush/drush": "^8.1"
+        "drush/drush": "^8.1 || ^9@beta"
```
Comment (zaporylie) [inline composer.json]: Let's only support for ^9@beta here. If someone needs support for Drush:8,  ^0.2 is the way to go. That also means - remove old command definition.

File: drush.services.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+services:
```
Comment (zaporylie) [inline drush.services.yml]: Do we need this file?

File: src/Commands/DrupalSettingsCommands.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?php
```
Comment (zaporylie) [inline src/Commands/DrupalSettingsCommands.php]: This file should be deleted in favour of src/Commands/DrupalSettingsCommands.php?

File: src/Commands/drupal_settings/DrupalSettingsCommands.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Drush\Commands\drupal_settings;
+
+use Drupal\Core\DrupalKernel;
```
Comment (zaporylie) [inline src/Commands/drupal_settings/DrupalSettingsCommands.php]: This is unused.

File: src/Commands/drupal_settings/DrupalSettingsCommands.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+use Drupal\Core\Site\Settings;
+use Drush\Commands\DrushCommands;
+use Symfony\Component\HttpFoundation\Request;
```
Comment (zaporylie) [inline src/Commands/drupal_settings/DrupalSettingsCommands.php]: This is unused.

File: src/Commands/drupal_settings/DrupalSettingsCommands.php [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ *
+ * In addition to a commandfile like this one, you need a drush.services.yml
+ * in root of your module.
+ *
+ * See these files for an example of injecting Drupal services:
+ *   - http://cgit.drupalcode.org/devel/tree/src/Commands/DevelCommands.php
+ *   - http://cgit.drupalcode.org/devel/tree/drush.services.yml
+ */
+class DrupalSettingsCommands extends DrushCommands {
```
Comment (zaporylie) [inline src/Commands/drupal_settings/DrupalSettingsCommands.php]: Maybe remove `Drupal` part and leave simple `SettingsCommands`?

**Review by zaporylie — CHANGES_REQUESTED** [review]
Left some inline comments.

**Pull request closed without merging**